pkg/clusterversion/cockroach_versions.go

```diff
@@ -260,6 +265,10 @@ var versionsSingleton = keyedVersions{
 		Key:     Start22_1,
 		Version: roachpb.Version{Major: 21, Minor: 2, Internal: 2},
 	},
+	{
+		Key:     TargetBytesAvoidExcess,
+		Version: roachpb.Version{Major: 21, Minor: 2, Internal: 4},
```

Comment:
Whoops, sorry, not sure why I swept this one up with all the 21.1 versions. Thanks for catching it!